minvws/nl-kat-coordination
# Remove old Findings Report

### Changes

This PR fixes the old "Download PDF" button on the Findings page, which redirected to the old OOI Report. 
This report doesn't exist anymore in the new flow.

We decided to delete the button completely, until we've created the new report flow. It just doesn't make sense to put a "Generate Findings Report" here, which redirects you to the Generate Report flow with ALL ooi's selected for the Finding Report type. And currently there's no easy way to not select ALL objects. 

What this PR does:
- Remove the button on the Findings Page
- Remove `ooi_report.py`, `ooi_report.html` and `ooi_report_404.html`
- Remove the corresponding URLs
- Remove the corresponding tests
- Remove mock from conftest.py
- Add "Source" to new Findings Report
- Change the input_ooi_types in FindingsReport (it used to be ALL_TYPES, but now excludes some types)

### Issue link
Closes #3500

### Demo
![afbeelding](https://github.com/user-attachments/assets/37270413-93f2-411f-91ce-cb2a81dce91e)


### QA notes
- Go to Findings page
- "Download PDF" button should be removed
- Generate the report

### New tickets
#3578 

---

### Code Checklist



- [ ] I have written unit tests for the changes or fixes I made. // Removed redundant unit tests




---

## Checklist for code reviewers:

Copy-paste the checklist from [the docs/source/templates folder](https://github.com/minvws/nl-kat-coordination/blob/main/docs/source/templates/pull_request_template_review_code.md) into your comment.

---

## Checklist for QA:

Copy-paste the checklist from [the docs/source/templates folder](https://github.com/minvws/nl-kat-coordination/blob/main/docs/source/templates/pull_request_template_review_qa.md) into your comment.
